examples/hif2a/fit_to_multiple_rbfes.py

```diff
@@ -74,22 +74,10 @@
 #   and a "training configuration"
 #       (which describes the overall training loop)
 
-# don't make assumptions about working directory
 
 # locations relative to project root
 root = Path(__file__).absolute().parent.parent.parent
```

Comment:
Not really for this PR, but realized reading through this on my own, was that we probably want to do something like...

```
import timemachine
root = Path(timemachine.__file__).parent
```

This way this file can move and the implication of root stays the same